<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyProfile</class>
 <widget class="QMainWindow" name="MyProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 31, 31);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item alignment="Qt::AlignLeft|Qt::AlignTop">
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
	margin-top:10px;
	margin-left:10px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
            </property>
            <property name="text">
             <string>GO TO HOME</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item alignment="Qt::AlignHCenter|Qt::AlignTop">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>30</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>PROFILE</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item alignment="Qt::AlignRight|Qt::AlignTop">
           <widget class="QPushButton" name="pushButton_goToPage">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
	margin-top:10px;
	margin-left:10px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
            </property>
            <property name="text">
             <string>GO TO MY PAGE</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,4,8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,9">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <pointsize>18</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>USER ID : </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_userId">
                  <property name="font">
                   <font>
                    <pointsize>18</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>USER00</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,9">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_16">
                <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
                 <widget class="QLabel" name="label_3">
                  <property name="font">
                   <font>
                    <pointsize>18</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>NAME : </string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QLabel" name="label_name">
                  <property name="font">
                   <font>
                    <pointsize>18</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 255, 255);</string>
                  </property>
                  <property name="text">
                   <string>Name</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,3">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                   <widget class="QLabel" name="label_5">
                    <property name="font">
                     <font>
                      <pointsize>16</pointsize>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string>FRIENDS LIST</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_13">
                  <item>
                   <widget class="QListWidget" name="listWidget_friends">
                    <property name="font">
                     <font>
                      <family>Arial</family>
                      <pointsize>-1</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QListWidget {
    background-color: #333333;
    border: 1px solid #666666;
    color: #CCCCCC;
    font-family: Arial;
    font-size: 12px;
}

QListWidget::item {
    background-color: #444444;
    padding: 8px;
}

QListWidget::item:hover {
    background-color: #555555;
}

QListWidget::item:selected {
    background-color: #007ACC;
    color: #FFFFFF;
}

QListWidget::item:selected:!active {
    background-color: #007ACC;
    color: #FFFFFF;
}

QListWidget::item:selected:active {
    background-color: #005C99;
}
</string>
                    </property>
                    <property name="itemAlignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_14" stretch="1,3">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_17">
                  <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                   <widget class="QLabel" name="label_6">
                    <property name="font">
                     <font>
                      <pointsize>16</pointsize>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string>LIKED PAGES</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_15">
                  <item>
                   <widget class="QListWidget" name="listWidget_likedPages">
                    <property name="styleSheet">
                     <string notr="true">QListWidget {
    background-color: #333333;
    border: 1px solid #666666;
    color: #CCCCCC;
    font-family: Arial;
    font-size: 12px;
}

QListWidget::item {
    background-color: #444444;
    padding: 8px;
}

QListWidget::item:hover {
    background-color: #555555;
}

QListWidget::item:selected {
    background-color: #007ACC;
    color: #FFFFFF;
}

QListWidget::item:selected:!active {
    background-color: #007ACC;
    color: #FFFFFF;
}

QListWidget::item:selected:active {
    background-color: #005C99;
}
</string>
                    </property>
                    <property name="itemAlignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_19">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_19">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_21">
                <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                 <widget class="QPushButton" name="pushButton_addPost">
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
                  </property>
                  <property name="text">
                   <string>ADD POST</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_22">
                <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                 <widget class="QPushButton" name="pushButton_addActivity">
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
                  </property>
                  <property name="text">
                   <string>ADD ACTIVITY</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_17">
                <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                 <widget class="QPushButton" name="pushButton_memory">
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
                  </property>
                  <property name="text">
                   <string>MEMORY</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_20">
                <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                 <widget class="QPushButton" name="pushButton_addCreatePage">
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
                  </property>
                  <property name="text">
                   <string>CREATE A PAGE</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_18" stretch="5,2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_21">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="10">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_25">
                  <item>
                   <widget class="QLabel" name="label_Post">
                    <property name="font">
                     <font>
                      <pointsize>20</pointsize>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: rgb(255, 255, 255);
background-color: #333333;</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_20">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_26" stretch="1,0,1,1,1,1,0,1">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_28">
                  <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
                   <widget class="QPushButton" name="pushButton_previousPost">
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
                    </property>
                    <property name="text">
                     <string>PREVIOUS POST</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item alignment="Qt::AlignRight|Qt::AlignVCenter">
                   <widget class="QPushButton" name="pushButton_like">
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
                    </property>
                    <property name="text">
                     <string>LIKE</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_29">
                  <item>
                   <widget class="QLabel" name="label_likesCount">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string>LIKES : </string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item alignment="Qt::AlignRight|Qt::AlignVCenter">
                   <widget class="QPushButton" name="pushButton_addComment">
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
                    </property>
                    <property name="text">
                     <string>COMMENT</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_30">
                  <item>
                   <widget class="QLabel" name="label_commentsCount">
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string>COMMENTS :</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_27">
                  <item alignment="Qt::AlignRight|Qt::AlignVCenter">
                   <widget class="QPushButton" name="pushButton_nextPost">
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
    border: none;
    border-radius: 10px;
    background-color: #333333;
    color: white;
    padding: 10px 20px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    outline: none;
}

QPushButton:hover {
    background-color: #555555;
}</string>
                    </property>
                    <property name="text">
                     <string>NEXT POST</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
